addon/components/horizontal-list-swiper/gesture/component.js

```diff
@@ -6,22 +6,34 @@ import Ember from 'ember';
 import RecognizerMixin from 'ember-gestures/mixins/recognizers';
 import layout from './template';
 import { run } from '@ember/runloop';
+import {getOwner} from '@ember/application';
 
 const {
   String
 } = Ember;
 
 export default Component.extend(RecognizerMixin, {
-
+  window: window,
```

Comment:
`window,`

Reply:
yeah ok. Can we not make that a blocker please? (fixed in next PR)